docs/mp/metrics/micrometer.adoc

```diff
@@ -16,18 +16,56 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 
-= Micrometer Metrics
+= Micrometer Support
 :description: Helidon Micrometer integration
 :keywords: micrometer, helidon, metrics, integration, microprofile
 :rootdir: {docdir}/../..
 
 include::{rootdir}/includes/mp.adoc[]
 
-include::{rootdir}/includes/metrics/micrometer-shared.adoc[tags=intro]
-include::{rootdir}/includes/metrics/micrometer-shared.adoc[tags=prereq]
+== Contents
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<Usage, Usage>>
+- <<API, API>>
+- <<Configuration, Configuration>>
+- <<Examples, Examples>>
+- <<Additional Information, Additional Information>>
 
-== Using Micrometer in Your Application
-Add the Micrometer `@Timed` and `@Counted` annotations to methods in your application.
+include::{rootdir}/includes/metrics/micrometer-shared.adoc[tag=intro]
+include::{rootdir}/includes/metrics/micrometer-shared.adoc[tag=prereq]
+
+== Usage
+
+=== Registering and Updating Meters
+To use Micrometer support, you can simply add the Micrometer `@Timed` and `@Counted` annotations to methods in your application. Helidon automatically registers those meters with the Micrometer composite `MeterRegistry`.
+
+In addition to annotating your methods, your code
+include::{rootdir}/includes/metrics/micrometer-shared.adoc[tag=use-micrometer-api]
+
+
+include::{rootdir}/includes/metrics/micrometer-shared.adoc[tag=accessing-endpoint-intro]
+
+
+
+
+== API
```

Comment:
provide some context about available APIs (options, usage?) We typically try to include some text in the H1/H2 sections. Not required but sometimes easier to navigate.

Reply:
Fixed